README.md

```diff
@@ -29,17 +31,17 @@ Tests have not yet been created for this repository.
 
 ### Usage
 
-To run file converter:
+To run sequencing file converter:
 ```
-python <command here> 
+python seqfileconverter -f <input file> -o <output file> -d <discard file> -m <metadata file> -t <file type> 
 ```
```

Comment:
Can you add the line `python seqfileconverter -h` and also the output of that? And is it `seqfileconverter` or `seqfileconverter.py`?